frontend/app/src/components/table-filter/SuggestedItem.vue

```diff
@@ -9,13 +10,28 @@ const props = withDefaults(
   defineProps<{
     suggestion: Suggestion;
     chip?: boolean;
+    editMode?: boolean;
   }>(),
   {
     chip: false,
+    editMode: false,
   },
 );
 
-const { suggestion } = toRefs(props);
+const emit = defineEmits<{
+  (e: 'cancel-edit'): void;
+  (e: 'update:search', value: string): void;
+}>();
```

Comment:
```suggestion
const emit = defineEmits<{
  'cancel-edit': [];
  'update:search': [value: string];
}>();
```

this is most a nitpick to use the literal format instead, unfortunately it does not have an autofix so we have to slowly do it by hand